src/Contexts/NetworkManagerContext.tsx

```diff
@@ -89,6 +89,15 @@ interface NetworkManagerContext {
 
   setTransferTxHash: (input: string) => void;
   transferTxHash: string;
+
+  setDepositRecipient: (input: string | undefined) => void;
+  depositRecipient: string | undefined;
+
+  setDepositAmount: (input: number | undefined) => void;
```

Comment:
When it can be `undefined`?

Reply:
Will check it

Reply:
We should use `undefined` here as in the [`resetDeposit` method](https://github.com/Cerebellum-Network/chainbridge-ui/blob/ec1583be38c4909b3bcba91660b9880e19eddf71/src/Contexts/ChainbridgeContext.tsx#L107) we use `undefined` value.